generators/exercises/intergalactic_transmission.py

```diff
@@ -0,0 +1,60 @@
+FUNC_PROTO = """\
+#include "vendor/unity.h"
+
+#include <stdint.h>
+
+#define BUFFER_SIZE 100
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+#define WRONG_PARITY -1
+
+extern int transmit_sequence(uint8_t *buffer, const uint8_t *message, int message_length);
+extern int decode_message(uint8_t *buffer, const uint8_t *message, int message_length);
+"""
+
+
+def describe(case):
+    description = case["description"]
+    property = case["property"]
+    return f"{property} {description}"
+
+
+def array_literal(digits):
+    return ", ".join(digits).replace("'", "")
+
+
+def gen_func_body(prop, inp, expected):
+    message = inp["message"]
+
+    str_list = []
+    if len(message) == 0:
+        message = "NULL"
+        message_length = 0
+    else:
+        str_list.append("uint8_t message[] = {\n")
+        str_list.append("    // clang-format off\n")
+        str_list.append(f"    {array_literal(message)}\n")
+        str_list.append("    // clang-format on\n")
```

Comment:
I don't see any meaningful structure in the formatting here, so I'd rather have clang-format do the line breaks here:
```suggestion
        str_list.append(f"    {array_literal(message)}\n")
```